edb/pgsql/metaschema.py

```diff
@@ -5949,8 +5952,16 @@ def _generate_sql_information_schema() -> List[dbops.Command]:
             '0'::cid                            AS cmax,
             NULL                                AS ctid
         FROM (
-            SELECT DISTINCT schema_name, module_id
+            SELECT schema_name, module_id
```

Comment:
Can this wind up with something duplicated?

Reply:
No, I use `UNION` just below (and UNION removed all duplicates, I've tested that)

Reply:
:+1: